packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -97,6 +101,14 @@ export async function createNewFluidFile(
 
     const odspUrl = createOdspUrl({... newFileInfo, itemId, dataStorePath: "/"});
     const resolver = new OdspDriverUrlResolver();
+
+    // caching the converted summary
+    const snapshot: IOdspSnapshot = convertSummaryTreeToIOdspSnapshot(createNewSummary);
+    const value: ISnapshotCacheValue = { snapshot, sequenceNumber: 0 };
```

Comment:
Any chance we do not hard-code sequence number here, and instead extract it from snapshot itself?
It is easy to track where it can be found - take a look at detachedContainerRefSeqNumber in container - it is stored in .attributes blob. We already parse attributes blobs in other places in driver, maybe worth extracting it into separate API.
Scanning source code, I see getDocAttributesFromProtocolSummary().

Same applies to convertSummaryTreeToIOdspSnapshot() - it encodes zero in there, it should not.

Reply:
ok! I will use import  getDocAttributesFromProtocolSummary()